compose-examples/prysm-only/docker-compose.yaml

```diff
@@ -96,4 +97,10 @@ services:
       - ./data/grafana:/var/lib/grafana
     <<: *logging
 
+  node-exporter:
+    image: prom/node-exporter:latest
+    hostname: node-exporter
+    ports:
+      - 127.0.0.1:9100:9100
```

Comment:
This port bind should not be necessary. Prometheus accesses node-exporter directly in docker internal networking.

Reply:
port binding is removed (127.0.0.1:9100:9100)